src/Package/Impl/DataInspect/REnvironment.cs

```diff
@@ -40,5 +40,8 @@ public REnvironment(string name)
         public string EnvironmentExpression { get; }
 
         public REnvironmentKind Kind { get; }
+
+        // For Accessibility: screen reader reads values returned by 'ToString()'
+        public override string ToString() => Name ?? base.ToString();
```

Comment:
I'd rather add `AutomationProperties.Name` to the `EnvironmentComboBox.ItemContainerStyle` style